### ISSUE TITLE
Various pages have text not color contrast accessible

### ISSUE THREAD

**Issue by elioqoshi** [OpenTechFund/hypha#1222]
The header image on the Results & Learnings page is so bright, that basic WCAG color contrast standards are not met.

![image](https://user-images.githubusercontent.com/5436686/57773088-79078580-7717-11e9-9ea8-d5ae34d7ebea.png)

https://www.opentech.fund/results/

The ratio is 1.43:1 at some points. (4.5 is suggested) for WCAG AA.

This can be fixed by choosing a darker header image. Maybe a darker blurred out version of something like this: https://unsplash.com/photos/8RX3W79_UTE

**Comment by elioqoshi** [comment]
Same with other pages:
https://www.opentech.fund/labs/
https://www.opentech.fund/about/
and even the landing page hero.

A good solution is using the same imagery, but adding a layer effect on top of it>

Some content uses colors which don't reach WCAG AA as well:
![image](https://user-images.githubusercontent.com/5436686/57773974-99384400-7719-11e9-9e2a-e77475ea8444.png)

**Comment by elioqoshi** [comment]
@frjo @dmcdev1 this would be great to be addressed in tandem with the new page designs. Happy to assist with this.

**Comment by frjo** [comment]
@elioqoshi We should definitely fix this. 

Background images it not something we developers control but things like the light red text for deadlines definitely is. 

Will replace that with a darker red. 

Are there other parts that have problems as well? 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by elioqoshi** [comment]
> Background images it not something we developers control 

I was actually thinking the images could be converted to grayscale and overlaid with a the OTF blue with a screen layer effect. This should be doable via CSS even. 

> Will replace that with a darker red.

I'd suggest use a larger framework to follow a system rather =fixing single bugs. But for the time being that should fix it.

> Are there other parts that have problems as well?

Probably. I only had a quick color contrast look. The site would benefit from a generic accessibility audit which might be more work.
